api/test/wfl/tools/parallel_runner.clj

```diff
@@ -0,0 +1,25 @@
+(ns wfl.tools.parallel-runner
+  (:require [clojure.test :as test])
+  (:import (java.io OutputStreamWriter StringWriter OutputStream)))
+
+(defn -main
+  [& args]
+  (letfn [(go! [& tests]
+            (let [null   (new OutputStreamWriter (OutputStream/nullOutputStream)) ;; Java 11+
```

Comment:
I don't think it is worth it to try to roll our own from scratch to back-port this and it is only available in Java 11 and higher. I think we try to keep compatibility with 8 so maybe I just put another stringwriter here?